### PR TITLE
[MyIP] add local Ip

### PR DESCRIPTION
**Pull request by kkange**
## Description

[myip] I updated the extension.

1. Added local-ip list items.
2. Updated the raycast api version.

## Screencast

스크린샷 2022-03-17 14 25 38

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed in the `metadata` folder

### CODE REVIEW

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @kkange 👋 

Thanks for the update 🔥 Everything works as expected 💪 

We're now adding support for [categories](https://developers.raycast.com/basics/prepare-an-extension-for-store#categories), [screenshots](https://developers.raycast.com/basics/prepare-an-extension-for-store#screenshots) and [changelog](https://developers.raycast.com/basics/prepare-an-extension-for-store#version-history) – do you have time for adding this too

**Comment by kkange** [comment]
Hi @pernielsentikaer 

I added new things.

However, the method of making screenshots is very difficult.

File: extensions/myip/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# MyIP Changelog
+
+​
+
+## [Added Additions] - 2022-03-17
```
Comment (pernielsentikaer) [inline extensions/myip/CHANGELOG.md:5]:
```suggestion
# MyIP Changelog

## [Added Additions] - 2022-03-17
```
Seems like you have a hidden char on line three and some extra spaces, could you remove it? (So first entry starts on line 3 instead?

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @kkange 👋 

Thanks for taking your time adding it, much appreciated!

We'll be looking into finding an easier way to make the screenshots, but for now, this is the way, and it seems like you done it the right way, awesome! 🔥 

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Comment by kkange** [comment]
There was a mistake in the file and it was corrected.

**Pull request closed without merging**